bundles/org.openhab.binding.http/README.md

```diff
@@ -183,10 +185,34 @@ http://www.domain.org/home/lights/23871/?status=OFF&date=2020-07-06
 
 ```java
 Thing http:url:foo "Foo" [
-    baseURL="https://example.com/api/v1/metadata-api/web/metadata", 
+    baseURL="https://example.com/api/v1/metadata-api/web/metadata",
+    headers="key1=value1", "key2=value2", "key3=value3",
+    refresh=15] {
+        Channels:
+            Type string : text "Text" [ stateTransformation="JSONPATH($.metadata.data)" ]
+}
+```
+
+Multiple transformations can be chained in the .things file by listing them as comma-separated values:
+
+```java
+Thing http:url:foo "Foo" [
+    baseURL="https://example.com/api/v1/metadata-api/web/metadata",
+    headers="key1=value1", "key2=value2", "key3=value3",
+    refresh=15] {
+        Channels:
+            Type string : text "Text" [ stateTransformation="REGEX(.*meta.*)", "JSONPATH($.metadata.data)" ]
```

Comment:
Is this syntax also working if you configure `stateTransformation` and `commandTransformation`?

Reply:
Yes, although I haven't personally tested this on an actual installation.

It should look like this:

```java
  Type string : text "Text" [ 
    stateTransformation="REGEX(.*meta.*)", "JSONPATH($.metadata.data)",
    commandTransformation="REGEX(.*meta.*)", "JSONPATH($.metadata.data)"
   ]
```